extensions/blocks/gif/editor.scss

```diff
@@ -7,6 +7,9 @@
 	.components-base-control__field {
 		text-align: center;
 	}
+	.components-placeholder__label svg {
+		margin-right: 1ch;
```

Comment:
I think this looks much better, but we should make the change in core, not here - all the blocks suffer from this lack of spacing.

Reply:
Happy to make this change in core, but I think we should keep it in here until that change has merged.